package.json

```diff
@@ -1,42 +1,57 @@
 {
   "name": "tree-sitter-kotlin",
   "version": "0.3.7",
-  "description": "Tree-sitter grammar for Kotlin",
+  "description": "Kotlin grammar for tree-sitter",
+  "repository": "github:fwcd/tree-sitter-kotlin",
+  "author": "fwcd",
+  "license": "MIT",
   "main": "bindings/node",
-  "scripts": {
-    "test": "tree-sitter test",
-    "install": "node-gyp rebuild",
-    "generate": "tree-sitter generate",
-    "parse": "tree-sitter parse",
-    "build-wasm": "tree-sitter build-wasm",
-    "playground": "tree-sitter playground"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/fwcd/tree-sitter-kotlin.git"
-  },
+  "types": "bindings/node",
   "keywords": [
+    "incremental",
+    "parsing",
     "tree-sitter",
-    "kotlin",
-    "grammar"
+    "kotlin"
+  ],
+  "files": [
+    "grammar.js",
+    "binding.gyp",
+    "prebuilds/**",
+    "bindings/node/*",
+    "queries/*",
+    "src/**",
+    "*.wasm"
   ],
-  "author": "fwcd",
-  "license": "MIT",
-  "gypfile": true,
-  "bugs": {
-    "url": "https://github.com/fwcd/tree-sitter-kotlin/issues"
-  },
-  "homepage": "https://github.com/fwcd/tree-sitter-kotlin#readme",
   "dependencies": {
-    "nan": "^2.19.0"
+    "node-addon-api": "^8.0.0",
+    "node-gyp-build": "^4.8.1"
   },
   "devDependencies": {
-    "tree-sitter-cli": "^0.22.1"
+    "prebuildify": "^6.0.1",
+    "tree-sitter-cli": "^0.22.6"
+  },
+  "peerDependencies": {
+    "tree-sitter": "^0.21.1"
+  },
+  "peerDependenciesMeta": {
+    "tree-sitter": {
+      "optional": true
+    }
+  },
+  "scripts": {
+    "install": "node-gyp-build",
+    "prestart": "tree-sitter build --wasm",
+    "start": "tree-sitter playground",
+    "test": "node --test bindings/node/*_test.js"
```

Comment:
This changes a few scripts, I assume this is the convention that upstream recommends now? In that case, we should probably update the instructions in the README too.

Reply:
I found the old names to be a bit more descriptive, e.g. `npm run playground` is more obvious to me than `start` (and building the Wasm blob might be something one wants to do separately as well, so I'd prefer not to bury it in a prestart script).

Ideally I would like to encourage contributors to use the npm-installed version of tree-sitter to make sure the build environments are somewhat consistent. Using a separate version of `tree-sitter` in CI and removing the descriptive scripts feel like a step in the opposite direction.

Reply:
`npm start` is a standard npm command and it makes sense in this case.
`npm run prestart` and `npm start --ignore-scripts` can also be used separately.

Reply:
I understand, but `npm run prestart` is even less obvious to me than `npm run build-wasm`, this looks like a technicality/an implementation detail that is not meant to be invoked separately.

Reply:
How often do you need to build a Wasm binary, outside of CI, without then starting the playground?

Reply:
Not that often, but now and then to test the grammar in different contexts. Automation also needs this.

Reply:
I've extracted this to #132, which implements `start` and `prestart` by delegating to the descriptive scripts.